Update String, RegExp, DataTime.js

Saida2s opened the pull request.

Saida2s committed:
Update String, RegExp, DataTime.js

Saida2s committed:
Update String, RegExp, DataTime.js

Saida2s committed:
Update String, RegExp, DataTime.js

YauhenEp commented on String, RegExp, DataTime.js:
```diff
@@ -1,33 +1,32 @@
 
+//...............1...................
+var str = 'ahb acb aeb aeeb adcb axeb'.replace(/a\w\b\s/);;
+console.log(str)
```
No, you need to use Regexp. This is the task for Regexp. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp

YauhenEp commented on String, RegExp, DataTime.js:
```diff
@@ -1,33 +1,32 @@
 
 
+let text='ahb acb aeb aeeb adcb axeb'
+console.log(text.slice(0,11))
```
No, use Regexp

YauhenEp commented on String, RegExp, DataTime.js:
```diff
@@ -1,33 +1,32 @@
 
-let firstMatch = matchAll[0];
-alert( firstMatch[0] );
+//...............2....................
```
good

YauhenEp commented on String, RegExp, DataTime.js:
```diff
@@ -1,33 +1,32 @@
+var day = (new Date()).getDate();
+console.log(day);
+
+var month = (new Date()).getMonth();
```
this line return 3, now is April this is forth month.

Saida2s committed:
Update String, RegExp, DataTime.js

in the first task i wrote to expressions. I think two of them correct, but second one is not working.
 Can you text me where  is problem?

Saida2s commented:
Good evening Sir,

I've done all the tasks. Could you please go through?

Thank you in advance,

Saida

пт, 22 апр. 2022 г., 15:31 YauhenEp ***@***.***>:

> ***@***.**** approved this pull request.
> ------------------------------
>
> In String, RegExp, DataTime.js
> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856070345>:
>
> >
> +//...............1...................
> +var str = 'ahb acb aeb aeeb adcb axeb'.replace(/a\w\b\s/);;
> +console.log(str)
>
> No, you need to use Regexp. This is the task for Regexp.
>
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
> ------------------------------
>
> In String, RegExp, DataTime.js
> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856070494>:
>
> >
>
> +let text='ahb acb aeb aeeb adcb axeb'
> +console.log(text.slice(0,11))
>
> No, use Regexp
> ------------------------------
>
> In String, RegExp, DataTime.js
> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856071072>:
>
> >
> -let firstMatch = matchAll[0];
> -alert( firstMatch[0] );
> +//...............2....................
>
> good
> ------------------------------
>
> In String, RegExp, DataTime.js
> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856073333>:
>
> >
>  //...............3....................
> -let date = new Date();
> +var day = (new Date()).getDate();
> +console.log(day);
> +
> +var month = (new Date()).getMonth();
>
> this line return 3, now is April this is forth month.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Saida2s/Javascript/pull/21#pullrequestreview-949857797>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYGMQI7EDIAANVSMQYJAZIDVGJ5XNANCNFSM5TWPA4HA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

Saida2s commented:
Good evening,

Please can you check my tasks?




Best wishes,
Saida

ср, 27 апр. 2022 г., 22:05 Saida Ismoilova ***@***.***>:

> Good evening Sir,
>
> I've done all the tasks. Could you please go through?
>
> Thank you in advance,
>
> Saida
>
> пт, 22 апр. 2022 г., 15:31 YauhenEp ***@***.***>:
>
>> ***@***.**** approved this pull request.
>> ------------------------------
>>
>> In String, RegExp, DataTime.js
>> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856070345>:
>>
>> >
>> +//...............1...................
>> +var str = 'ahb acb aeb aeeb adcb axeb'.replace(/a\w\b\s/);;
>> +console.log(str)
>>
>> No, you need to use Regexp. This is the task for Regexp.
>>
>> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
>> ------------------------------
>>
>> In String, RegExp, DataTime.js
>> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856070494>:
>>
>> >
>>
>> +let text='ahb acb aeb aeeb adcb axeb'
>> +console.log(text.slice(0,11))
>>
>> No, use Regexp
>> ------------------------------
>>
>> In String, RegExp, DataTime.js
>> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856071072>:
>>
>> >
>> -let firstMatch = matchAll[0];
>> -alert( firstMatch[0] );
>> +//...............2....................
>>
>> good
>> ------------------------------
>>
>> In String, RegExp, DataTime.js
>> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856073333>:
>>
>> >
>>  //...............3....................
>> -let date = new Date();
>> +var day = (new Date()).getDate();
>> +console.log(day);
>> +
>> +var month = (new Date()).getMonth();
>>
>> this line return 3, now is April this is forth month.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Saida2s/Javascript/pull/21#pullrequestreview-949857797>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AYGMQI7EDIAANVSMQYJAZIDVGJ5XNANCNFSM5TWPA4HA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

YauhenEp commented:
Yes, I check your tasks today.





YAUHEN BIALETSKI
Senior Software Test Automation Engineer

Office:  +375 17 389 0100x39700<tel:+375%2017%20389%200100;ext=x39700>    Cell:  +375 29 688 1127<tel:+375%2029%20688%201127>
Email:  ***@***.******@***.***>
Microsoft Teams:  Call On ***@***.***>

Kuprevicha, 1/1
Minsk, Belarus
Timezone: (UTC+03:00)
EPAM.COM<http://www.epam.com/>
***@***.***

From: Saida2s ***@***.***>
Sent: Tuesday, May 10, 2022 5:47 PM
To: Saida2s/Javascript ***@***.***>
Cc: Yauhen Bialetski ***@***.***>; Comment ***@***.***>
Subject: Re: [Saida2s/Javascript] Update String, RegExp, DataTime.js (PR #21)

Good evening,

Please can you check my tasks?




Best wishes,
Saida

ср, 27 апр. 2022 г., 22:05 Saida Ismoilova ***@***.***>:

> Good evening Sir,
>
> I've done all the tasks. Could you please go through?
>
> Thank you in advance,
>
> Saida
>
> пт, 22 апр. 2022 г., 15:31 YauhenEp ***@***.***>:
>
>> ***@***.**** approved this pull request.
>> ------------------------------
>>
>> In String, RegExp, DataTime.js
>> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856070345><https://urldefense.com/v3/__https:/github.com/Saida2s/Javascript/pull/21*discussion_r856070345*3E__;IyU!!GF_29dbcQIUBPA!zbyBw71C7R3ox78HAA-adRcZZ9JLArZSsN7maOS6esL40ZuBJ38BUdy02G8IEjFLoK-0RdgtGvL1a9rYTbN5ighf62WHPg$>:
>>
>> >
>> +//...............1...................
>> +var str = 'ahb acb aeb aeeb adcb axeb'.replace(/a\w\b\s/);;
>> +console.log(str)
>>
>> No, you need to use Regexp. This is the task for Regexp.
>>
>> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp<https://urldefense.com/v3/__https:/developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp__;!!GF_29dbcQIUBPA!zbyBw71C7R3ox78HAA-adRcZZ9JLArZSsN7maOS6esL40ZuBJ38BUdy02G8IEjFLoK-0RdgtGvL1a9rYTbN5iggVfduecg$>
>> ------------------------------
>>
>> In String, RegExp, DataTime.js
>> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856070494><https://urldefense.com/v3/__https:/github.com/Saida2s/Javascript/pull/21*discussion_r856070494*3E__;IyU!!GF_29dbcQIUBPA!zbyBw71C7R3ox78HAA-adRcZZ9JLArZSsN7maOS6esL40ZuBJ38BUdy02G8IEjFLoK-0RdgtGvL1a9rYTbN5igiJZqZxRQ$>:
>>
>> >
>>
>> +let text='ahb acb aeb aeeb adcb axeb'
>> +console.log(text.slice(0,11))
>>
>> No, use Regexp
>> ------------------------------
>>
>> In String, RegExp, DataTime.js
>> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856071072><https://urldefense.com/v3/__https:/github.com/Saida2s/Javascript/pull/21*discussion_r856071072*3E__;IyU!!GF_29dbcQIUBPA!zbyBw71C7R3ox78HAA-adRcZZ9JLArZSsN7maOS6esL40ZuBJ38BUdy02G8IEjFLoK-0RdgtGvL1a9rYTbN5igjkIZ96hg$>:
>>
>> >
>> -let firstMatch = matchAll[0];
>> -alert( firstMatch[0] );
>> +//...............2....................
>>
>> good
>> ------------------------------
>>
>> In String, RegExp, DataTime.js
>> <https://github.com/Saida2s/Javascript/pull/21#discussion_r856073333><https://urldefense.com/v3/__https:/github.com/Saida2s/Javascript/pull/21*discussion_r856073333*3E__;IyU!!GF_29dbcQIUBPA!zbyBw71C7R3ox78HAA-adRcZZ9JLArZSsN7maOS6esL40ZuBJ38BUdy02G8IEjFLoK-0RdgtGvL1a9rYTbN5igjxTlpqBw$>:
>>
>> >
>> //...............3....................
>> -let date = new Date();
>> +var day = (new Date()).getDate();
>> +console.log(day);
>> +
>> +var month = (new Date()).getMonth();
>>
>> this line return 3, now is April this is forth month.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Saida2s/Javascript/pull/21#pullrequestreview-949857797><https://urldefense.com/v3/__https:/github.com/Saida2s/Javascript/pull/21*pullrequestreview-949857797*3E__;IyU!!GF_29dbcQIUBPA!zbyBw71C7R3ox78HAA-adRcZZ9JLArZSsN7maOS6esL40ZuBJ38BUdy02G8IEjFLoK-0RdgtGvL1a9rYTbN5igjEET81SA$>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AYGMQI7EDIAANVSMQYJAZIDVGJ5XNANCNFSM5TWPA4HA><https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AYGMQI7EDIAANVSMQYJAZIDVGJ5XNANCNFSM5TWPA4HA*3E__;JQ!!GF_29dbcQIUBPA!zbyBw71C7R3ox78HAA-adRcZZ9JLArZSsN7maOS6esL40ZuBJ38BUdy02G8IEjFLoK-0RdgtGvL1a9rYTbN5iggbzrL3WQ$>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>


—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/Saida2s/Javascript/pull/21*issuecomment-1122495190__;Iw!!GF_29dbcQIUBPA!zbyBw71C7R3ox78HAA-adRcZZ9JLArZSsN7maOS6esL40ZuBJ38BUdy02G8IEjFLoK-0RdgtGvL1a9rYTbN5iggZ1LLmBw$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/ARUUJMGQPELBVBCKKOAR6FTVJJZFLANCNFSM5TWPA4HA__;!!GF_29dbcQIUBPA!zbyBw71C7R3ox78HAA-adRcZZ9JLArZSsN7maOS6esL40ZuBJ38BUdy02G8IEjFLoK-0RdgtGvL1a9rYTbN5ighm5O1zTw$>.
You are receiving this because you commented.Message ID: ***@***.***>
